src/components/radio-button/radio-button.mdx

```diff
@@ -54,29 +54,12 @@ A legend can be set for the group, and each radio button can have a label.
 
 <Canvas of={RadioButtonStories.WithLegendAndLabels} />
 
-### With inline legend
```

Comment:
We still have the WithInlineLegend and EnableAdaptiveBehaviour stories in the tsx file, if we're satisfied this behaviour is adequately covered elsewhere and they aren't imported in another file we should remove them. We do need to ensure that any regressions are captured so it will need to either be unit/playwright/chromatic that does it as until the old layout is deprecated we still need to ensure it works etc

Reply:
Same needs confirming in all the stories we've updated, I agree the first thing consumers see should be the latest layout designs etc but we still have some who work on legacy projects so we can't not have something in the docs relating to that etc